moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -102,9 +97,11 @@ void ChompOptimizer::initialize()
   collision_detection::CollisionRequest req;
   collision_detection::CollisionResult res;
   req.group_name = planning_group_;
-  ros::WallTime wt = ros::WallTime::now();
+  // TODO
+  // ros::WallTime wt = ros::WallTime::now();
```

Comment:
No need to comment this, I think using clock from `std::chrono` should be sufficient 
```cpp
const auto wt = std::chrono::high_resolution_clock::now(); // Or system_clock
...
RCLCPP_INFO(LOGGER,"First coll check took " << std::chrono::duration<double>(std::chrono::high_resolution_clock::now() - wt).count());
```

Reply:
Fixed. I also put the runtime measurements at the end of `optimize()` back in (around line 523).